### PR TITLE
Finish Mobile version

### PR DESCRIPTION
**Pull request by Kajal19-del**
The goal of this project is to build a straightforward, responsive, single-page portfolio that will hold all of the past personal projects. You must configure the HTML-CSS linter for this project in accordance with the instructions provided in the HTML-CSS setup instructions and build the website with mobile users in mind.

# In this project I did the following activities:
- I create the repository with name "first-portfolio"
- I get help from Figma to complete this design
- I create the HTML, CSS, the README, gitignore and some other files.
- I add linters to the repository
- I add all the images need to complete this project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -51,6 +49,164 @@ <h1 class="header-title">I'm Kajal Pramanik to see you!</h1>
...
+
+      <ul class="section-skills">
+        <li class="skills">html</li>
+        <li class="skills">css</li>
+        <li class="skills">javascript</li>
+      </ul>
+
+      <a href="#" class="btn-text">See Project</a>
+
+    </section>
```
Comment (Ibizugbe) [inline index.html:142]: I think your projects should be wrapped inside a  `<section>` as required by [this current milestone](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), you should have 2 sections (the work and about section) this should be wrapped in the "work section". kindly make these changes👍😉🤗

File: index.html [hunk 2]
```diff
@@ -51,6 +49,164 @@ <h1 class="header-title">I'm Kajal Pramanik to see you!</h1>
...
+        <h3 class="skill-language">Frameworks</h3>
+        <i class="arrow right"></i>
+      </div>
+
+      <div class="skill-lang bottom-line">
+        <h3 class="skill-language">Skills</h3>
+        <i class="arrow right"></i>
+      </div>
+    </section>
+  </footer>
```
Comment (Ibizugbe) [inline index.html]: As described earlier, this should be wrapped in a section tag too, this is meant to be the about section😉.
you can also take away the `<footer>` tag as this is not the footer of the webpage😉 kindly make these changes😉🤗

File: index.html [hunk 3]
```diff
@@ -51,6 +49,164 @@ <h1 class="header-title">I'm Kajal Pramanik to see you!</h1>
...
+
+      <a href="#" class="btn-text">See Project</a>
+
+    </section>
+  </main>
+
+  <footer class="footer-section">
+    <section class="about">
+
+      <h1 class="title">About <br> Myself</h1>
```
Comment (Ibizugbe) [inline index.html]: I think for semantic purposes, a `<h2>` tag will be preferable here. it is more semantically correct considering that we already introduced the page with a `<h1>` tag🤗 these differences will be appreciated by screen readers😉 kindly implement the change.

File: index.html [hunk 4]
```diff
@@ -51,6 +49,164 @@ <h1 class="header-title">I'm Kajal Pramanik to see you!</h1>
...
+          <li>
+            <a href="#"><img src="./images/github.svg" alt="github"></a>
+          </li>
+          <li>
+            <a href="#"><img src="./images/angellist.svg" alt="angellist"></a>
+          </li>
+        </ul>
+      </div>
+
+      <a href="#" class="btn-text">Get my resume</a>
```
Comment (Ibizugbe) [inline index.html]: I think your about section has a little bit different from [the design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=poHBiyU0shXbvM5C-0).  and it is a  [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) that your project looks like the design.

this is the design😉
about desugn

this is your project😉
your about sec

kindly apply the right padding, margin, font size, color, and weight where applicable.🤗🤗🤗

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @Kajal19-del 👋, 

Good job so far!👍👍
There are some issues that you still need to work on to go to the next project but you are almost there!

198009982-d99c2507-f20d-4c00-90a3-a54097598b36


## Highlights
- Pull request has a descriptive heading and summary👍
- Project is professionally documented👍
- Commit messages are descriptive and meaningful👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Kajal19-del) [reply index.html:142]: @Ibizugbe I have alittle bit confusion, it's already wrapped inside a main tag with a class name(container-main). Should I just change the class name with (work-section). 🤗🤗🤗

please reply me, so I can update my code as soon as possible🙂🤝

Comment (bienvenuushindi) [reply index.html:142]: @Kajal19-del  I think this page has one main section and two parent sections. So the structure of this part of the page  should look like This  :arrow_down: 

```
<main class="container-main">
     <section id="work">
       <!--- work content here--->
    </section>
    <section id="about">
       <!--- about content here--->
    </section>
</main>
```

**Review by bienvenuushindi — APPROVED** [review]
# Approved 👏  

## Hi @Kajal19-del   👋, 

Well done implementing the requested changes 👏

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @bienvenuushindi  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**